crates/holochain_persistence_lmdb/src/eav/lmdb.rs

```diff
@@ -17,7 +16,7 @@ use std::{
 };
 use uuid::Uuid;
 
-const EAV_BUCKET: &str = "EAV";
+pub const EAV_BUCKET: &str = "EAV";
```

Comment:
why?

Reply:
Using the constant to generate a database name in in the `txn` module. I think I have to make a change which also makes this not necessary anyhow. Will leave this conversation unresolved until then.